application/ledger_query_service/src/ethereum/queries.rs

```diff
@@ -26,6 +27,89 @@ pub struct EthereumTransactionQuery {
     transaction_data_length: Option<usize>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct EthereumTransactionLogQuery {
+    logs: Vec<Vec<H256>>,
```

Comment:
:thinking: Should we provide a more user-friendly way here? Like, just `String`s and convert it to the hash ourselves? Might be easier to consume.

Also, these things are called `Topics` in Ethereum if I am not mistaken. I think it would be good to stick to the terminology.

Reply:
I'd like to preserve the safety of only accepting valid hashes. How can I do that if I allow any string?